_posts/2025-10-24-Making-Search-Smarter-System-Generated-Search-Pipelines.markdown

```diff
@@ -0,0 +1,342 @@
+---
+layout: post
+title: "Making search smarter with system-generated search pipelines"
+layout: post
+authors:
+   - bzhangam
+date: 2025-10-24
+has_science_table: true
+categories:
+   - technical-posts
+meta_keywords: system-generated search pipeline, OpenSearch 3.3, plugin development, system-generated pipeline, system-generated search processor
+meta_description: Learn how OpenSearch automatically generates and executes search processors during query evaluation, how plugin developers can extend this mechanism, and how to monitor system-generated processors using the Search Pipeline Stats API.
```

Comment:
Updated blog meta: 

**meta_keywords:** system-generated search pipelines, OpenSearch 3.3, search processors, search automation, search request processing, search pipeline API, search optimization, search intelligence, vector search, neural search, search monitoring

**meta_description:** Discover how the system-generated search pipelines in OpenSearch 3.3 automate search request processing, simplify plugin development, and enhance search intelligence without manual configuration